app/controllers/coronavirus_landing_page_controller.rb

```diff
@@ -29,11 +29,11 @@ def hub
     set_expiry 5.minutes
 
     @content_item = content_item.to_hash
-    breadcrumbs = [{ title: "Home", url: "/" }]
+    breadcrumbs = [{ title: t("shared.breadcrumbs_home"), url: "/" }]
     title = {
       text: @content_item.dig("details", "page_title") || @content_item["title"],
       context: {
-        text: "Coronavirus (COVID-19)",
+        text: t("corona_virus_landing_page.title"),
```

Comment:
```suggestion
        text: t("coronavirus_landing_page.title"),
```
Could we keep the key name consistent with the controller name?